src/nodedef.cpp

```diff
@@ -590,6 +590,14 @@ void ContentFeatures::deSerialize(std::istream &is, u16 protocol_version)
 	post_effect_color = readARGB8(is);
 	leveled = readU8(is);
 
+	// raillike drawtype: For old clients, fall back to
```

Comment:
The server holds all node data and uses `serialize()` to send it to the client, hence the code should be put into that function instead.

Reply:
The location of the code is correct but the comment is wrong, this is to accommodate old _servers_.